tests/unit/test_auth.py

```diff
@@ -3,24 +3,16 @@
 from unittest import mock
 
 import responses
-from earthaccess.auth import Auth
+from earthaccess import Auth
 
 
 class TestCreateAuth(unittest.TestCase):
-    @mock.patch("builtins.input")
-    @mock.patch("getpass.getpass")
-    def test_create_auth_wrong_credentials(self, user_input, user_password) -> bool:
-        user_input.return_value = "user"
-        user_password.return_value = "pass"
-        auth = Auth().login(strategy="interactive")
-        self.assertEqual(auth.authenticated, False)
```

Comment:
Do we not want to test a bad credentials case? Or is that effectively done in the tests below where `auth.authenticated = False`?

Reply:
I just added a case for that, to test that when we get a 401 response we are not authenticated.